analyzers/src/SonarAnalyzer.VisualBasic/SonarAnalyzer.VisualBasic.csproj

```diff
@@ -30,6 +30,18 @@
     <DefineConstants>TRACE;VB</DefineConstants>
   </PropertyGroup>
 
+  <Target Name="SignDlls" Condition="'$(SignAssembly)' == 'true' And '$(TargetFramework)' == 'NET46' " BeforeTargets="CopyBinaries">
```

Comment:
I think the `BeforeTargets` from the VSIX worked only by accident and can break at any point. IMO we should use this instead.
```
<Target Name="CopyBinaries" AfterTargets="Build" DependsOnTargets="SignDlls" Condition=" '$(TargetFramework)' == 'NET46' ">
```
Because in theory, msbuild can process these 3 targets in this order:
* SignDlls (will fail)
* Build
* CopyBinaries

Reply:
The ` BeforeTargets="CopyBinaries"` should be removed now, the `DependsOnTargets` takes care of it